packages/desktop-gui/src/project-nav/browsers.jsx

```diff
@@ -29,6 +36,12 @@ export default class Browsers extends Component {
     )
   }
 
+  _browsersFeature () {
+    return this.feature('browsers')
+    .map(() => (<div>Browsers:</div>))
+    .getOrElse()
```

Comment:
I know this will result in an undefined for a `Nothing`, but I think this should be  explicit (`getOrElse(null)`) so there is no ambiguity for anybody unfamiliar with using a `Maybe`.

Reply:
I personally find this API horrific to actually read and understand.

`.map` to me iterates over a collection and returns a new collection. Does not seem that it's doing that here. I get what it's doing, but why `map`? shouldn't it be like `.has` or maybe just `.if`?

`.getOrElse` is also confusing. Get or else what? Should this just be `.else` ? Why is it `.getOr*`?